File: practice/java/NumberPositiveNegative.java

import java.util.Scanner;

/**
 * @berif this function check number is positive or negative
 * @param number parameter isliye likha ki hame dekhana hai number negative hai
 * ya positive
 */
class numberPositiveNegative {

    static int positiveNegative(double number) {

        if (number < 0) {
            System.out.println("number is negative");
        } else if (number == 0) {
            System.out.println("number is zero");
        } else {
            System.out.println("number is positive");
        }
        return 0;
    }

    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a number");
        double number = sc.nextDouble();
        sc.close();
        positiveNegative(number);

    }

}

File: practice/programiz/number_positive_negative.c
#include <stdio.h>
/**
 * @brief This function checks if a given number is positive, negative, or zero.
 * @param number  parameter ka isliye likha hai kiyu hame ye dikhna number positive hai ya negative
 * @return int
 */

int number_positive_negative(int number)
{
    if (number < 0)
    {
        printf("number is negative\n");
    }
    else if (number == 0)
    {
        printf("number is zero\n");
    }
    else
    {
        printf("number is positive");
    }
    return 0;
}

int main(void)
{
    int number;
    printf("enter a number:\n");
    scanf("%i", &number);
    number_positive_negative(number);
    return 0;
}
